Gemfile

```diff
@@ -2,4 +2,8 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
-gemspec :name => "logstash-core"
+gem "logstash-core", "2.0.0.dev", :path => "."
+gem "ci_reporter", "1.9.3", :group => :development
```

Comment:
I agree ```ci_reporter``` should be in here. Another questions is what to do with rspecs and the coverage deps, I see then living in the gemspec, because all gem specific dependencies are there. I explain myself, If I want to run test I want test+coverage, but not to use ci_reports.

If finally all this deps are in the Gemfile, devutils have to be here too as it's mostlly all things for testing.

- purbon